spec/system/candidate_interface/candidate_accepts_offer_spec.rb

```diff
@@ -100,4 +104,21 @@ def and_i_see_that_i_declined_the_other_offer
       expect(page).to have_content 'Declined'
     end
   end
+
+  def when_i_have_another_application_without_offer
+    @application_choice_without_offer = create(
```

Comment:
You can reuse `@application_choice` here because it's also in a state that means you shouldn't be able to see the page.